js/main.js

```diff
@@ -0,0 +1,27 @@
+var $img = document.querySelector('.journal-image');
+var $title = document.querySelector('#title');
+var $photoURL = document.querySelector('#img-url');
+var $notes = document.querySelector('#notes');
+var $form = document.querySelector('.entry-form');
+
+function updatePhoto(event) {
+  $img.setAttribute('src', $photoURL.value);
+}
+
+function saveEntry(event) {
+  event.preventDefault();
+  var newEntry = {};
+  newEntry.title = $title.value;
+  newEntry.photoURL = $photoURL.value;
+  newEntry.notes = $notes.value;
+  newEntry.EntryID = data.nextEntryId;
+  data.nextEntryId++;
+  data.entries.push(newEntry);
```

Comment:
You want to add to the beginning of the array, not the end.

Reply:
Changed from push method to the unshift method!

[add to front of array demo.webm](https://user-images.githubusercontent.com/109567499/189770999-486fd063-9114-4471-8493-d9d44018ed35.webm)